examples/react/custom-links/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "tanstack-router-react-example-custom-links",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite --port=3001",
+    "build": "vite build",
+    "serve": "vite preview",
+    "start": "vite"
+  },
+  "dependencies": {
+    "@tanstack/react-query": "^5.17.8",
```

Comment:
~~Is there any reason `@tanstack/react-query` is being brought in for this example?~~

Edit: Ignore the comment, just realized this was copied over from our *basic* example.